doc/classes/Navigation.xml

```diff
@@ -28,6 +28,46 @@
 				Returns the path between two given points. Points are in local coordinate space. If [code]optimize[/code] is [code]true[/code] (the default), the agent properties associated with each [NavigationMesh] (radius, height, etc.) are considered in the path calculation, otherwise they are ignored.
 			</description>
 		</method>
+		<method name="get_closest_point_to_segment" qualifiers="const">
+			<return type="Vector3">
+			</return>
+			<argument index="0" name="from" type="Vector3">
+			</argument>
+			<argument index="1" name="to" type="Vector3">
+			</argument>
+			<argument index="2" name="use_collision" type="bool" default="false">
+			</argument>
+			<description>
+				Returns the closest point between the navigation surface and the segment.
+			</description>
+		</method>
+		<method name="get_closest_point" qualifiers="const">
+			<return type="Vector3">
+			</return>
+			<argument index="0" name="point" type="Vector3">
+			</argument>
+			<description>
+				Returns the nearest point on the navigation mesh surface nearest to the submitted point.
```

Comment:
"nearest point ... nearest to the submitted point" sounds unclear.

Reply:
Maybe "Returns the point closest to the provided [code]point[/code] on the navigation mesh surface."

Reply:
done